ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -170,13 +174,20 @@ type ${project.artifactId}_unitfile_config_systemd_exec = {
     'LimitRTTIME' ? long(-1..) # Specifies a limit (in microseconds) on the amount of CPU time that a process scheduled under a real-time scheduling policy may consume without making a blocking system call.
     'LimitSIGPENDING' ? long(-1..) # Specifies the limit on the number of signals that may be queued for the real user ID of the calling process.
     'LimitSTACK' ? long(-1..) # The maximum size of the process stack, in bytes.
+    'LogsDirectoryMode' ? string # octal notation
+    'LogsDirectory' ? string
     'Nice' ? long(-20..19)
     'OOMScoreAdjust' ? long(-1000..1000)
     'PrivateTmp' ? boolean
     'RootDirectory' ? string
+    'RuntimeDirectoryMode' ? string # octal notation
+    'RuntimeDirectoryPreserve' ? boolean
```

Comment:
> Takes a boolean argument or restart. If set to no (the default),

So perhaps better as a string with validation? Otherwise you can't set it to the value of "restart" :(

Reply:
```suggestion
    'RuntimeDirectoryPreserve' ? choice('yes', 'no', 'restart')
```